tfplot/wrapper.py

```diff
@@ -0,0 +1,307 @@
+''' Main plot operations. '''
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import functools
+import re
+
+from . import figure
+from . import util
+from .ops import plot, plot_many
+from .util import merge_kwargs
+
+from biwrap import biwrap
+
+from matplotlib.figure import Figure
+from matplotlib.axes import Axes
+
+
+# a dummy marker object to avoid pylint issues on decorators.
+REQUIRED = object()
+
+
+@biwrap
+def wrap(plot_func=REQUIRED, _sentinel=None,
```

Comment:
Why not using keyword only arguments? Compatibility?

Reply:
I want to support python 2.7 for the time being (See TF's cross entropy loss function).